pkg/gui/controllers/helpers/suggestions_helper.go

```diff
@@ -153,10 +153,22 @@ func (self *SuggestionsHelper) getRemoteBranchNames(separator string) []string {
 	})
 }
 
+func (self *SuggestionsHelper) getRemoteBranchNamesWithoutRemotePrefix() []string {
```

Comment:
The implementation is very similar to `getRemoteBranchNames()`, but I haven't been able to come up with a nice way of reducing the code duplication. Hints welcome :)

Reply:
I'm not fussed about that duplication. I can imagine the two methods evolving in separate ways.

Reply:
Function would not be added at all / be removed again when going with the new approach (select remote first, branch second)